RInAppMessaging/Classes/CampaignDispatcher.swift

```diff
@@ -44,9 +44,13 @@ internal class CampaignDispatcher: CampaignDispatcherType, TaskSchedulable {
     }
 
     func resetQueue() {
-        isDispatching = false
-        scheduledTask?.cancel()
```

Comment:
Why were these moved into the async q?

Reply:
Because I think it wasn't really necessary to keep those outside, and now it's more thread safe.